Calibration/HcalCalibAlgos/test/GammaJetAnalysis.cc

```diff
@@ -591,10 +591,6 @@ void GammaJetAnalysis::analyze(const edm::Event& iEvent, const edm::EventSetup&
 
       edm::Ref<reco::PhotonCollection> photonRef(photons, photon_tag.idx());
       HERE(Form("got photon ref, photon_tag.idx()=%d", photon_tag.idx()));
-#ifdef EDM_ML_DEBUG
-      edm::LogVerbatim("GammaJetAnalysis")
-          << "loosePhotonQual->at(photon_tag.idx())=" << loosePhotonQual->at(photon_tag.idx());
-#endif
```

Comment:
@bsunanda , this code was initially commented out but your PR https://github.com/cms-sw/cmssw/pull/34737 enabled it and placed it under `EDM_ML_DEBUG`. Looks like this code does not compile, please suggest the correct code to replace it or we can just remove it